### ISSUE TITLE
HuggingFaceToken doesn't work for Midjourney Human Verify

### ISSUE THREAD

**Issue by hoantran-it** [erictik/midjourney-api#278]
**Describe the bug**
I encounter this issue even though I have configured HuggingFaceToken. The request cannot be run on MJ due to a bot challenge, as shown in the screenshot.

**Expected behavior**
Does HuggingFaceToken still work for this case, or did I miss something? Thank you!

**Screenshots**
Midjourney Human Verify

![mj-bot-challenge](https://github.com/erictik/midjourney-api/assets/13161875/8cbe3542-5319-4956-b352-e138541c3582)

**Comment by winsonxu** [comment]
me too, 
Now as soon as it starts, it will be blocked within a day

**Comment by DmitryGolovin-azur** [comment]
use websocket. it works for me

**Comment by hoantran-it** [comment]
> just use websocket. It works for me

Yes, it works, but occasionally this challenge pops up, and the program can no longer be executed. Have you ever encountered this challenge? Or do you monitor for it and see if it's resolved using a Hugging Face token?

**Comment by DmitryGolovin-azur** [comment]
I don't have such issues, when I use websocket. Yes, I monitor when I need to enter a captcha, but there was no such case.

But when I don't use websocket, a captcha popup appears quite quickly.

I guess it is related to implementation: 
https://github.com/erictik/midjourney-api/blob/1346be21baa852938e5eaed4329bf72f2c45e8aa/src/midjourney.ts#L61

Midjourney don't like special symbols (like '['), try to avoid it.

**Comment by hoantran-it** [comment]
> I don't have such issues, when I use websocket. Yes, I monitor when I need to enter a captcha, but there was no such case.
> 
> When I don't use websocket, a captcha popup appears quite quickly.
> 
> I guess it is related to the implementation:
> 
> https://github.com/erictik/midjourney-api/blob/1346be21baa852938e5eaed4329bf72f2c45e8aa/src/midjourney.ts#L61
> 
> Midjourney don't like special symbols (like '['), try to avoid it.

Thank you for clarification!

**Comment by soberpdf** [comment]
I completed the human verification, but a few hours later, my account got suspended.

**Comment by soberpdf** [comment]
![dis](https://github.com/user-attachments/assets/fdeddab8-bf60-47f8-b5d7-2a592cb48b97)
After comparison, I found that there are some differences in parameter passing between this project and using Discord for parameter passing, as shown in the figure. This is an important indicator for monitoring human-computer interaction.

**Comment by soberpdf** [comment]
> > <img alt="dis" width="18.716659545898438" height="16.866668701171875" src="https://private-user-images.githubusercontent.com/119870221/361089973-fdeddab8-bf60-47f8-b5d7-2a592cb48b97.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ2MDQ5ODUsIm5iZiI6MTcyNDYwNDY4NSwicGF0aCI6Ii8xMTk4NzAyMjEvMzYxMDg5OTczLWZkZWRkYWI4LWJmNjAtNDdmOC1iNWQ3LTJhNTkyY2I0OGI5Ny5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgyNVQxNjUxMjVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hZDc5MjU5ODUwYjAxMDg4NmFkMGMxZjk4YTE3ODNjZDA0M2U4YWIyZDY5MjExZmNlZDUxNmUzZWRmNWJmZDVjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.yXl_nEGcNrqbLc66u3AwgUiYeevPpyMrlrlILkNo4Bw"> After comparison, I found that there are some differences in parameter passing between this project and using Discord for parameter passing, as shown in the figure. This is an important indicator for monitoring human-computer interaction.在进行对比后，我注意到这个项目与使用 Discord 进行参数传递在参数传递方式上存在一些不同，具体如图所示。这表明了人机交互的一个关键监测点。
> 
> Does it work？ 做得出来吗？

yes
